pdc/apps/common/renderers.py

```diff
@@ -248,6 +248,7 @@ def get_filters(view):
     FILTERS_CACHE[view] = filters
     return filters
 
+
```

Comment:
yes, if we don't add the blank line; can't pass when run  "make flake8" in github. https://travis-ci.org/product-definition-center/product-definition-center/builds/177633056 .